### ISSUE TITLE
Presets for server creation

### ISSUE THREAD

**Issue by ItIsSeven** [pterodactyl/panel#2059]
Presets for 1.0, when creating servers so everything is pre-filled for you (CPU limits/RAM/Disk space etc...)
Should be relatively easy to implement, and makes the process a lot quicker to install a new duplicate/similar server. Especially useful for those who rely solely on the panel and don't use any 3rd party options (WHMCS/Ptero API) when creating servers

For someone who creates and deletes servers a lot, and needs multiple running at times, it's a bit tedious filling in everything when maybe only one small thing needs to be changed

**Comment by FoksVHox** [comment]
Use the template when creating a feature request.

Also, how should that work?

**Comment by Sir3lit** [comment]
Im very curious why you'd be installing and deleting servers so often?

**Comment by qmtd0000** [comment]
> Use the template when creating a feature request.
> 
> Also, how should that work?

My bad on not using the template, I looked at a few other feature requests and it looked like they were accepted without having followed the template.

When it comes to the the issue, I'm not sure how exactly would be the way you'd want to implement it. I just want something that works really. Was thinking something simple where you fill in everything, and save it as a preset, and then once the presets saved you can (in the future when creating a new server) use the preset you saved and it'll fill in everything on the page.

> Im very curious why you'd be installing and deleting servers so often?

The panel is used in a lot of different ways, I'm not the only one that creates servers on a regular basis and has to go through the process of filling everything in again because there's no autofill/preset option with the panel. So this would just be a basic and simple feature that allows people to create duplicate/similar servers very quickly.

Thank you

**Comment by snaildos** [comment]
Not just this, it can also limit external bots, it would be great to set a limit, so we cannot create 10 of the same templates.

**Comment by hwalker928** [comment]
(If this this classified as promo, feel free to remove this comment)
I have made an addon which does this, which can be found here on the Pterodactyl Discord:
https://discord.com/channels/122900397965705216/574327246546796544/844292830267441182

**Comment by snaildos** [comment]
> (If this this classified as promo, feel free to remove this comment)
> I have made an addon which does this, which can be found here on the Pterodactyl Discord:
> https://discord.com/channels/122900397965705216/574327246546796544/844292830267441182

Yes, This isn't really a "preset" tho. It would be cool to have a link you can click on, then authorize your account and it creates the values, and copies the files from another server.

**Comment by ozanaaslan** [comment]
Will this be added?

**Comment by FoksVHox** [comment]
If it isn't closed, it still planned

**Comment by ozanaaslan** [comment]
yay

**Comment by Upd4ting** [comment]
Planned for which release and when?

**Comment by snaildos** [comment]
Would presets include files?
It be cool if we make our own server, configure everything and turn into a preset.

**Comment by DaneEveritt** [comment]
In the interest of maintainer sanity and cleaning up an enormous backlog of stale feature requests we have moved to requiring all feature requests be discussed within [Discussions](https://github.com/orgs/pterodactyl/discussions/categories/feature-requests). Please consider re-opening this request there if you believe it is still relevant and worth consideration.

See https://github.com/pterodactyl/panel/issues/5491 for additional context.

Thanks.

<sub>_This comment is in the interest of issue tracker maintenance: it does not imply that maintainers are endorsing or dismissing your request._</sub>
